src/containers/MovieInformationContainer.js

```diff
@@ -104,6 +106,14 @@ class MovieInformation extends Component {
       let trailers;
       if (this.props.currentMovie.videos.results.length === 0) {
         trailers = <div>No trailers to show</div>;
+      } else if (this.props.currentMovie.videos.results.length === 1) {
+        trailers = (
```

Comment:
It's probably time to extract the trailer part into its own component, since it has relatively complex behavior.